src/providers/comments-provider.tsx

```diff
@@ -42,14 +53,16 @@ export const CommentsProvider: React.VoidFunctionComponent<{
       effectIsStale = true;
     };
   }, [accidentId]);
+
   const providerValue = React.useMemo<CommentsContextValue>(
     () => ({
       newCommentText,
       setNewCommentText,
       comments,
+      commentsAreLoading: loading,
       setComments,
     }),
-    [newCommentText, setNewCommentText, comments, setComments],
+    [newCommentText, comments, loading],
```

Comment:
`setNewCommentText` is a setter from `React.useState`. Its reference does not change, so we don’t need to add this variable to hook dependencies. ESLint is happy with that.